dev/io.openliberty.microprofile.health.file.healthcheck.internal_fat/fat/src/io/openliberty/microprofile/health/file/healthcheck/fat/SimpleFileBasedHealthCheckTest.java

```diff
@@ -42,577 +40,233 @@
-         */
-        /*
-         * Checks that require to check that all files are created may encounter a scenario where FAT test is way ahead of the server.
-         * This results in the files not existing yet. isFilesCreated() will retry up to 2 seconds (w/ 250ms cycles).
-         */
-        Assert.assertTrue("Expected all files to be created: Review isAllHealthCheckFilesCreated logs for state of files.", FATSuite.isFilesCreated(serverRootDirFile));
-
-        /*
-         * The checkInterval is at 30 seconds.
-         * Due to slowness of server startup, or app startups or test execution startup we'll wait 12 and then 5 seconds.
-         * We will check the last 10 seconds and 5 seconds respectively for each cycle.
-         * We wait 12 and check the last 10 due the fact that the update phase maybe have issue the first health check queries during the wait.
-         * If that is the case, waiting 12 seconds and checking the last 12 seconds would fail due to the file being modified during that duration.
-         * and expect the live and ready files not to be updated.
-         *
-         * Then we'll wait 20 seconsd and we should expect it to have been updated during that time frame.
-         *
-         */
-        TimeUnit.SECONDS.sleep(12);
-        Assert.assertFalse(Constants.READY_SHOULD_NOT_HAVE_UPDATED,
-                           HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getReadyFile(serverRootDirFile), Duration.ofSeconds(10)));
-        Assert.assertFalse(Constants.LIVE_SHOULD_NOT_HAVE_UPDATED,
-                           HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getLiveFile(serverRootDirFile), Duration.ofSeconds(10)));
-
-        TimeUnit.SECONDS.sleep(5);
-        Assert.assertFalse(Constants.READY_SHOULD_NOT_HAVE_UPDATED,
-                           HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getReadyFile(serverRootDirFile), Duration.ofSeconds(5)));
-        Assert.assertFalse(Constants.LIVE_SHOULD_NOT_HAVE_UPDATED,
-                           HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getLiveFile(serverRootDirFile), Duration.ofSeconds(5)));
-
-        /*
-         * We are elapsed 38 seconds after we detected files are created (on the test infra).
-         * Checking within last 12 seconds. (i.e. elapsed after file create ~26-38).
-         * Big time window to account for slowness or "quickness" of the server.
-         */
-        TimeUnit.SECONDS.sleep(20);
-        Assert.assertTrue(Constants.READY_SHOULD_HAVE_UPDATED,
-                          HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getReadyFile(serverRootDirFile), Duration.ofSeconds(12)));
-        Assert.assertTrue(Constants.LIVE_SHOULD_HAVE_UPDATED,
-                          HealthFileUtils.isLastModifiedTimeWithinLast(HealthFileUtils.getLiveFile(serverRootDirFile), Duration.ofSeconds(12)));
+        Assert.assertTrue(Constants.STARTED_SHOULD_NOT_HAVE_CREATED,
```

Comment:
waitForFileMissing only checks until the file is missing. Meaning this passes immediately without checking that files aren't created after the duration.